src/gennylib/src/quiesce.cpp

```diff
@@ -36,7 +36,7 @@
 
 namespace genny {
 
-const int DROPPED_COLLECTION_RETRIES = 1000;
+const int DROPPED_COLLECTION_RETRIES = 20;
```

Comment:
I encountered a very strange behavior. With the previous value of 1000 and a sleep time of 1s, the quiesce actor test hangs. This happens even though running the QuiesceActor workload has no issues (they both do the same thing). 

When you lower the retry to 20 it ceases to hang. I've stared at this for a long time and cannot reason what is happening. 

Note that this behavior occurs even without my changes.